core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1562,6 +1562,20 @@ private CarbonCommonConstants() {
 
   public static final String CARBON_LUCENE_INDEX_STOP_WORDS_DEFAULT = "false";
 
+  //////////////////////////////////////////////////////////////////////////////////////////
+  // MV parameter start here
+  //////////////////////////////////////////////////////////////////////////////////////////
+
+  /**
+   * Property to enable MV rewrite
+   */
+  @CarbonProperty(dynamicConfigurable = true)
```

Comment:
You wanted to make this dynamic configurable, but forgot to add in 
SessionParams#validateKeyValue ?